qa/tasks/mgr/test_progress.py

```diff
@@ -149,6 +149,14 @@ def _osd_count(self):
         osd_map = self.mgr_cluster.mon_manager.get_osd_dump_json()
         return len(osd_map['osds'])
 
+    def _set_no_recovery_backfill(self, flag):
+        # Set recovery and backfill on or off
+        assert flag == 'set' or flag == 'unset'
```

Comment:
why not pass a boolean parameter? are we going to extend the `flag` more more capabilities to convey more messages to the callee than just `on` or `off`?

Reply:
you're right a boolean would have been better

Reply:
You're boolean makes more sense